docs/content/concepts/sui-architecture/consensus.mdx

```diff
@@ -1,41 +1,40 @@
 ---
 title: Consensus
-sidebar_label: Consensus 
-description: Overview of the Sui consensus.
+sidebar_label: Consensus
+description: Overview of the Sui consensus
 ---
 
-The basic purpose of consensus in blockchains is to agree on a consistent order and ensure the availability of transactions.
+The purpose of consensus in blockchains is to agree on a consistent order and ensure the availability of transactions.
 
-On Sui, consensus has a simple API: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
+On Sui, consensus has a simple API. Validators submit different user transactions to consensus concurrently. The consensus outputs a consistent stream of transactions across all well-behaving validators.
```

Comment:
```suggestion
On Sui, consensus has a straightforward API. Validators submit different user transactions to consensus concurrently. The consensus outputs a consistent stream of transactions across all well-behaving validators.
```
I always suggest avoiding the word "simple" (even though it wasn't you that used it here).